pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -814,7 +814,7 @@ type PoolSpec struct {
 
 	// DEPRECATED: use Parameters instead, e.g., Parameters["compression_mode"] = "force"
 	// The inline compression mode in Bluestore OSD to set to (options are: none, passive, aggressive, force)
-	// +kubebuilder:validation:Enum=none;passive;aggressive;force;""
+	// +kubebuilder:validation:Enum=none;passive;aggressive;force
```

Comment:
Why remove the empty string? Seems like we still need the empty option in case compression is not specified.

Reply:
It is not possible to configure compression_mode with an empty string ("").
```
$ kubectl -n rook-ceph exec -it deploy/rook-ceph-tools -- bash
bash-5.1$ ceph osd pool set replicapool1 compression_mode ""        
Error EINVAL: unrecognized compression mode ''
bash-5.1$ ceph osd pool set replicapool1 compression_mode force
set pool 2 compression_mode to force
bash-5.1$ 
```

Reply:
For backward compatibility we will need to keep the empty string. Hopefully my other comment will help fix the error reported.